metrics_proto.go

```diff
@@ -117,6 +119,19 @@ func (se *statsExporter) handleMetricsProtoUpload(payloads []*metricProtoPayload
 	return nil
 }
 
+// metricSignature creates a unique signature consisting of a
+// metric's type and its lexicographically sorted label values
```

Comment:
Just one last comment request. Since you've created the issue, please add it here to finally make this comment:
```go
// metricSignature creates a unique signature consisting of a
// metric's type and its lexicographically sorted label values.
// See https://github.com/census-ecosystem/opencensus-go-exporter-stackdriver/issues/120
func metricSignature(metric *googlemetricpb.Metric) string {
```